### DISCUSSION TITLE
How to disable automatic right shift after pressing "enter" in ITEM_INPUT function.

### DISCUSSION THREAD

**Discussion by AlissaAskerov** [forntoh/LcdMenu#145] [Q&A]
Hi there, I'm looking to make a very simple LCD interface without the need for left and right buttons (I only need to enter single characters and do not want to be able to move left/right and append more characters). I notice that currently after cycling through my defined charset and pressing enter, it automatically shifts to the right, looking for another char input. This isn't necessarily a problem since I can just exit this submenu, but if I were to want to enter this submenu again and edit, it automatically starts on that second char entry. Is there a way to disable this automatic shifting? I've looked through all the headers and can't seem to find it.

Thank you in advance!

**Comment by forntoh** [comment]
Currently, there's no way to disable it and I don't plan to do it anytime soon, feel free to fork the repo and create a PR, I'll be happy to review and merge.
